posts/2019/11/Files-Improvements.md

```diff
@@ -0,0 +1,77 @@
+<!--
+.. title: File management improvements in Spyder4
+.. slug: File-management-improvements-in-Spyder4
+.. date: 2019-11-01 12:00:00 UTC-05:00
+.. author: Juanita Gomez & Gonzalo Peña
+.. tags: Labs, Spyder
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+Spyder 4.0 is almost ready! It has been in the making for 1.5 years, and it contains lots of interesting new features. In this post we will focus on the files plugin where several improvements have been made regarding file management and interface. 
```

Comment:
```suggestion
Spyder 4.0 is almost ready! It has been in the making for well over a year and a half, and it contains lots of interesting new features. We will focus on the Files pane in this post, where we've made several improvements have been made regarding file management and interface. 
```

Reply:
* Spell out small numbers of years
* Prefer active voice, especially for blog posts
* The plugins are called "Panes" in the actual Spyder UI and are in sentence case, and calling them "plugins" (while accurate from a Spyder developer perspective) may confuse regular users into thinking they are optional add-ons, as that term usually refers to. Its also consistent with all the blogs, docs, UI text, guides, etc., that I wrote.
* Missing some commas

Also, @goanpeca or @ccordoba12 can confirm, but development on the Spyder 4 `master` branch was already well underway by the time I joined the team in the fall of 2017, well over 2 years ago, and I'm pretty sure quite a bit longer.